### PR TITLE
Pok 15 search UI

### LINKED ISSUE #15: POK-15- 🍿 Feature - feat: Add `SearchBar` UI
# Adds the UI for the `SearchBar` 

Image

_Pictured, the `SearchBar` and various sub-elements in different states_

The `SearchBar` UI is the main form control for searching for Pokemon via the PokeAPI. This form should have a text input that can be used to search for Pokemon either by `name` or `id`. This dynamic query is controller by the `select` element which has two options: `Name` or `Id`. The final element in the search bar is the stylized `Submit` button which executes the HTTP request against the PokeAPI using the information collected via the form. 

For this story only implement the UI. We will add the actual search functionality in a separate story.

### PR DESCRIPTION
**Pull request by mathiuskitchens**
# Description

<<_Please include a summary of the change._>>

## Type of change

- [ ] - fix: - patches a bug in the codebase
- [ ] - feat: - introduces a new feature
- [ ] - build: - relates to build tooling / project scaffolding
- [x] - chore: - non-code related changes, like adding packages / libraries
- [ ] - docs: - changes related to documentation

## Related Issue

Does this change have a related issue in the project?

- [x] - Yes - Closes #15 
- [ ] - No

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -78,7 +78,8 @@ summary,
 time,
 mark,
 audio,
-video {
+video,
```
Comment (wijohnst) [inline src/styles.css]: I wouldn't add my styling to the base elements. You should define a class like `search-input` or `pokedex-input` and style that. 

If we attach this specific styling to each default element, what happens if we need an default HTML `input` or `select`? In general, always try to build on top of something (extensibility) versus a tightly coupled change like this.

File: src/styles.css [hunk 2]
```diff
@@ -138,3 +139,48 @@ table {
...
+}
+
+select {
+  width: 200px;
+  height: 76px;
+  text-align: center;
+  color: var(--poke-black);
+}
+
+.go-button {
```
Comment (wijohnst) [inline src/styles.css:171]: This is much better!

**Review by wijohnst — COMMENTED** [review]
We should add `.DS_Store` to `.gitignore` and remove those changes.

File: src/styles.css [hunk 3]
```diff
@@ -138,3 +138,52 @@ table {
 body {
   background: #e15c6a;
 }
+
+#searchbar {
```
Comment (wijohnst) [inline src/styles.css]: You should add the styling here and take it off of `input` and `select`. You don't want to explicitly style the top-level `input` or `select` elements. 

We might want to talk about this in-person. Its a good opportunity to talk about some big coding ideas.

**Review by wijohnst — COMMENTED** [review]
Closer but still too tightly coupled.

**Comment by mathiuskitchens** [comment]
🙄

**Review by wijohnst — COMMENTED** [review]
LGTM!

**Review by wijohnst — APPROVED** [review]

**Pull request merged**